src/Sulu/Component/SmartContent/Orm/DataProviderRepositoryTrait.php

```diff
@@ -348,12 +370,10 @@ protected function appendDatasource($datasource, $includeSubFolders, QueryBuilde
     /**
      * Extension point to append order.
      *
-     * @param array $sortBy
+     * @param string $sortBy
      * @param string $sortMethod
      * @param string $alias
      * @param string $locale
-     *
-     * @return array parameters for query
      */
     protected function appendSortBy($sortBy, $sortMethod, QueryBuilder $queryBuilder, $alias, $locale)
```

Comment:
This function also contains some array specific code.

Reply:
I think you removed that in #5179